src/server/auth/cmds/cmds.go

```diff
@@ -154,24 +154,40 @@ Activate Pachyderm's auth system, and restrict access to existing data to the ro
 						Id:           clientId,
 						Name:         clientId,
 						TrustedPeers: trustedPeers,
-						RedirectUris: []string{redirect},
+						RedirectUris: []string{redirect, redirectPrivileged},
 					},
 				})
 				if err != nil {
 					return errors.Wrapf(grpcutil.ScrubGRPC(err), "failed to configure OIDC client ID")
 				}
-
 				if _, err := c.SetConfiguration(c.Ctx(),
-					&auth.SetConfigurationRequest{Configuration: &auth.OIDCConfig{
-						Issuer:          issuer,
-						ClientID:        clientId,
-						ClientSecret:    oidcClient.Client.Secret,
-						RedirectURI:     redirect,
-						LocalhostIssuer: true,
-						Scopes:          scopes,
-					}}); err != nil {
+					&auth.SetConfigurationRequest{
+						LoginType: auth.OIDCLoginType_STANDARD,
+						Configuration: &auth.OIDCConfig{
+							Issuer:          issuer,
+							ClientID:        clientId,
+							ClientSecret:    oidcClient.Client.Secret,
+							RedirectURI:     redirect,
+							LocalhostIssuer: true,
+							Scopes:          scopes,
+						}}); err != nil {
 					return errors.Wrapf(grpcutil.ScrubGRPC(err), "failed to configure OIDC in pachd")
 				}
+				// when activating auth for the enterprise server, also set a Configuration for it to redirect to a privileged oauth flow
+				if _, err := c.SetConfiguration(c.Ctx(),
+					&auth.SetConfigurationRequest{
+						LoginType: auth.OIDCLoginType_PRIVILEDGED,
+						Configuration: &auth.OIDCConfig{
+							Issuer:          issuer,
+							ClientID:        clientId,
+							ClientSecret:    oidcClient.Client.Secret,
+							RedirectURI:     redirectPrivileged, // who configures the privileged redirect? Extra param? Should we instead just get a privileged Domain?
```

Comment:
What's the goal of the privileged redirect versus the normal one? We have all the state server-side, do we need two different URIs?

Reply:
I see what you mean. I guess we could use the state to achieve this. 

My thought was that we didn't have a way to parameterize the HTTP call to Callback to pass the information of whether to mint the privileged ID Token. I see that the state could be that parameterization